neo3-gui/neo3-gui/ClientApp/src/components/Wallet/walletdetail.js

```diff
@@ -189,7 +189,7 @@ class Walletdetail extends React.Component {
                 )}
               />
               <div className="mb4 text-r">
-                <Button type="primary" onClick={this.showPrivate}>{t("button.show details")}</Button>
+                <Button type="primary" onClick={this.showPrivate} style={{ visibility: 'hidden' }}>{t("button.show details")}</Button>
```

Comment:
This will only being hidden for multi-signature address. 
For normal address, it needs to be displayed

Reply:
Fixed.